app/(auth)/register.js

```diff
@@ -95,6 +98,44 @@ export default function Register() {
 		}
 	};
 
+		const handlePresentModalPress = useCallback(() => {
+		bottomSheetModalRef.current?.present();
+	}, []);
+
+	const handleModalPress = (
+		modalContent,
+		snapPoints,
+		handleText,
+		handleColor,
+		extra = () => {}
+	) => {
+		extra();
+		setModalContent(modalContent);
+		setSnapPoints(snapPoints);
+		setHandles(handle(handleText, handleColor));
+		bottomSheetModalRef.current?.present();
+	};
+
+	const renderContent = () => {
+		switch (modalContent) {
+			case 'tos':
+				return (
+					<StyledView className='w-[90%] flex-1'>
+						<BottomSheetFlatList
+							data={[{ key: 'terms' }]}
+							renderItem={({ item }) => <Terms />}
+							keyExtractor={(item) => item.key}
+							showsVerticalScrollIndicator={false}
+						/>
+					</StyledView>
+				);
+			default:
+				return (
+					<></>
+				)			
+		}
+	};
+
```

Comment:
rather than a whole function, just add the code from `123-130` inside the `BottomSheetModal`

Reply:
This modal will be used in later PRs

Reply:
At least a Bug report text field, likely a request feature text field as well

Reply:
on the register page?

Reply:
I see, I was using this on my phone and thought was for the new page.

Reply:
Notably, this will be used in the future when moving the 'Take a Selfie!' old modal type to our new BottomSheetModal type